catroid/res/values-de/strings.xml

```diff
@@ -462,14 +462,14 @@
   <string name="phiro_motor_stop">Stop Phiro motor</string>
   <string name="brick_phiro_motor_forward_action">Move Phiro motor forward</string>
   <string name="brick_phiro_motor_backward_action">Move Phiro motor backward</string>
```

Comment:
Again, not everything is translated so far in German. That's why you see english strings.